.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
 
   test-build-golang-prev:
     docker:
-      - image: circleci/golang:1.12
+      - image: circleci/golang:1.13
```

Comment:
Ah, this is meant to compile for the previous release of go, since people might still be installing with `go get`. See https://github.com/segmentio/aws-okta/pull/243#issuecomment-552015513 for a workaround to the problem you were seeing compiling

Reply:
thanks, I read the comment after pushing the change. I'll revert the comment and use xerrors instead.